mne/report/report.py

```diff
@@ -2545,8 +2545,12 @@ def _render_one_bem_axis(self, *, mri_fname, surfaces,
 
         return html
 
-    def _render_raw_butterfly_segments(self, *, raw: BaseRaw, image_format,
-                                       tags):
+    def _render_raw_butterfly_segments(
+        self, *, raw: BaseRaw, image_format, tags
+    ):
+        orig_annotations = raw.annotations.copy()
+        raw.set_annotations(None)
```

Comment:
Since `raw` can be passed directly rather than via filename, this should really be wrapped in a `try/finally` (or equivalently, a `@contextmanager`) that ensures that, no matter what, `raw.set_annotations(orig_annotations` gets called

Reply:
(like if there is an error with plotting, or the user presses ctrl-C during plotting, etc.)

Reply:
I think the cleanest approach would be to add a new kwarg to plot_raw, but the signature is already so massive....

Reply:
It would be tricky there because of interactive plotting mode. I would assume YAGNI on that

Reply:
I've decided to go with `try/except` for now. LMK if you'd prefer a context manager instead (I felt it was a bit overkill but I'm open to revise this judgment :))